draft-ietf-quic-transport.md

```diff
@@ -240,178 +240,119 @@ x (*) ...
-unidirectional streams and bidirectional streams. Unidirectional streams always
-have this bit set to 1 and bidirectional streams have this bit set to 0.
-
-The two type bits from a Stream ID therefore identify streams as summarized in
-{{stream-id-types}}.
-
-| Low Bits | Stream Type                      |
-|:---------|:---------------------------------|
-| 0x0      | Client-Initiated, Bidirectional  |
-| 0x1      | Server-Initiated, Bidirectional  |
-| 0x2      | Client-Initiated, Unidirectional |
-| 0x3      | Server-Initiated, Unidirectional |
+stream.  Client-initiated streams have even-numbered Stream IDs (with the bit
+set to 0), and server-initiated streams have odd-numbered Stream IDs (with the
+bit set to 1).
+
+The second least significant bit (0x2) of the Stream ID distinguishes between
+bidirectional streams (with the bit set to 0) and unidirectional streams (with
+the bit set to 1).
+
+The least significant two bits from a Stream ID therefore identify a stream as
+one of four types, as summarized in {{stream-id-types}}.
+
+| Bits | Stream Type                      |
+|:-----|:---------------------------------|
+| 0x0  | Client-Initiated, Bidirectional  |
+| 0x1  | Server-Initiated, Bidirectional  |
+| 0x2  | Client-Initiated, Unidirectional |
+| 0x3  | Server-Initiated, Unidirectional |
 {: #stream-id-types title="Stream ID Types"}
 
-The first bidirectional stream opened by the client is stream 0.
-
-A QUIC endpoint MUST NOT reuse a Stream ID.  Streams of each type are created in
-numeric order.  Streams that are used out of order result in opening all
-lower-numbered streams of the same type in the same direction.
-
-
-## Stream Concurrency {#stream-concurrency}
-
-QUIC allows for an arbitrary number of streams to operate concurrently.  An
-endpoint limits the number of concurrently active incoming streams by limiting
-the number of streams of each type (see {{stream-limit-increment}}).
-
-The stream limit is specific to each endpoint and applies only to the peer that
-receives the setting.  That is, the client limits the number of streams of each
-type the server can initiate, and the server limits the number of streams the
-client can initiate.  Each endpoint can respond on streams initiated by the
-other peer, regardless of whether it is permitted to initiate new streams.
-
-Endpoints MUST NOT exceed the limit set by their peer.  An endpoint that
-receives a STREAM frame with an ID greater than the limit it has sent MUST treat
-this as a stream error of type STREAM_LIMIT_ERROR ({{error-handling}}).
-
-A receiver cannot renege on an advertisement; that is, once a receiver
-advertises a stream limit using the MAX_STREAMS frame, advertising a smaller
-limit has no effect.  A receiver MUST ignore any MAX_STREAMS frame that does
-not increase the stream limit.
+Within each type, streams are expected to be created with numerically increasing
+Stream IDs.  A Stream ID that is used out of order results in all streams of
```

Comment:
```suggestion
Stream IDs.  A stream opened an out of order Stream ID results in all streams of
```
(using a stream does not necessarily imply opening it, and therefore also not lower numbered streams)